src/js/components/charts/ChartStripes.js

```diff
@@ -15,7 +14,7 @@ var ChartStripes = React.createClass({
     var count = props.count;
     var width = props.width / (2 * count);
 
-    return _.map(_.range(0, count), function (i) {
+    return Object.keys(Array(count).fill()).map(function (i) {
```

Comment:
oh i see, you're called `Object.keys` on an array. i think that's really odd. plus, the keys would be strings so it's not really what we want.

why not just do `Array(count).map` and then use the index? (2nd argument)